bindings/wasm/lib/stardust_identity_client.ts

```diff
@@ -1,8 +1,7 @@
 // Copyright 2020-2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-// @ts-ignore: path is set to match runtime transpiled js path when bundled.
-import {IStardustIdentityClient, StardustDID, StardustDocument, StardustIdentityClientExt} from './identity_wasm';
+import {IStardustIdentityClient, StardustDID, StardustDocument, StardustIdentityClientExt} from 'identity_wasm';
 
 // NOTE: this import path is replaced with `/web` in the `build/web.js` script.
 import type {Client, INodeInfoWrapper, SecretManager} from '@cycraig/iota-client-wasm/node';
```

Comment:
Can we use the same path alias for the client import too?

Reply:
Will absolutely try!

Reply:
I tried `tsc-alias` with `paths` myself yesterday and didn't make much progress, but hopefully I was just missing something obvious.

Reply:
The repo for `ts-mocha` says it uses [tsconfig-paths](https://www.npmjs.com/package/tsconfig-paths) and to pass `--paths` for it to work? 

https://github.com/piotrwitek/ts-mocha

Should we use `tsconfig-paths` instead? https://npmtrends.com/tsc-alias-vs-tsconfig-paths-vs-tscpaths-vs-tspath

Reply:
My understanding is that `tsconfig-paths` works at runtime, while `ts-alias` fixes the paths in the artifacts. So we might actually need to use both 🙈